src/main/java/com/epam/ta/reportportal/core/project/impl/UpdateProjectHandlerImpl.java

```diff
@@ -104,6 +106,9 @@
 
 	private final ProjectConverter projectConverter;
 
+	@Autowired
+	private ObjectMapper objectMapper;
```

Comment:
can't find the usage of objectMapper here. why do we need it?

Reply:
removed